src/xenia/kernel/user_module.h

```diff
@@ -100,6 +101,7 @@ class UserModule : public XModule {
 
   std::string name_;
   std::string path_;
+  uint64_t hash_;
```

Comment:
Should initialize this to `0` or some other way to indicate that the value is invalid.